src/main/java/com/minecolonies/coremod/blocks/BlockMinecoloniesRack.java

```diff
@@ -213,14 +223,15 @@ public void neighborChanged(final IBlockState state, final World worldIn, final
 
     /**
      * This returns a complete list of items dropped from this block.
-     *
+     * @deprecated but we still need it.
      * @param world   The current world
      * @param pos     Block position in world
      * @param state   Current state
      * @param fortune Breakers fortune level
      * @return A ArrayList containing all items this block drops
      */
     @Override
+    @Deprecated
     public List<ItemStack> getDrops(final IBlockAccess world, final BlockPos pos, final IBlockState state, final int fortune)
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3AS1133)